src/Test/Utilities/Portable/Metadata/MetadataReaderUtils.cs

```diff
@@ -301,7 +323,10 @@ private static string DumpRec(this MetadataReader reader, EntityHandle handle)
                 case HandleKind.AssemblyReference:
                     return reader.GetString(reader.GetAssemblyReference((AssemblyReferenceHandle)handle).Name);
                 case HandleKind.TypeDefinition:
-                    return reader.GetString(reader.GetTypeDefinition((TypeDefinitionHandle)handle).Name);
+                    {
+                        var type = reader.GetTypeDefinition((TypeDefinitionHandle)handle);
```

Comment:
>var [](start = 24, length = 3)

Please spell out the type.

Reply:
This matches the code below.

---
In reply to: [194571582](https://github.com/dotnet/roslyn/pull/27589#discussion_r194571582) [](ancestors = 194571582)

Reply:
> This matches the code below.

Sorry, this doesn't make it easier to understand what the code is doing. this is a new code and it doesn't follow our team's coding style. I think the readers would appreciate if you change the code below too.

---
In reply to: [194615884](https://github.com/dotnet/roslyn/pull/27589#discussion_r194615884) [](ancestors = 194615884,194571582)